package.json

```diff
@@ -13,6 +14,14 @@
     "url": "git+https://github.com/Mudano/ui-react.git"
   },
   "author": "Mudano Ltd. (http://mudano.com)",
+  "contributors": [
+    "Andy Berry <andy.berry@mudano.com",
+    "Antoine Fromentin <antoine.fromentin@mudano.com>",
+    "David Corbacho <david.corbacho@mudano.com>",
+    "Jakub Frej <jakub.frej@mudano.com",
+    "Makis Spyratos <makis.spyratos@mudano.com>",
+    "Matt Davies <matt.davies@mudano.com>"
```

Comment:
Should these be our github-associated emails? Or perhaps `@users.noreply.github.com` to prevent spam like many people here choose to do https://github.com/facebook/react/blob/master/AUTHORS ?

Reply:
Perhaps lose the emails altogether?

Reply:
Or just put `"The Mudano UI React Team": "info@mudano.com"`

Reply:
Chatted with Andy on Slack, will remove